app/src/test/java/comp3350/mealbuddy/tests/objects/MealTest.java

```diff
@@ -0,0 +1,144 @@
+package comp3350.mealbuddy.tests.objects;
+
+import comp3350.mealbuddy.objects.consumables.FoodIntPair;
+import comp3350.mealbuddy.objects.consumables.Meal;
+import comp3350.mealbuddy.objects.consumables.Edible;
+import comp3350.mealbuddy.objects.consumables.Food;
+
+
+
+import org.junit.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MealTest {
+
+    public static final int NEGATIVEQUANTITY_TESTVALUE = 0;
```

Comment:
Why are you naming it negative it its 0???  Also have underline between Negative and quantity